sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -517,7 +590,9 @@ public ConnectionConfiguration withBearerToken(String bearerToken) {
     public ConnectionConfiguration withKeystorePath(String keystorePath) {
       checkArgument(keystorePath != null, "keystorePath can not be null");
       checkArgument(!keystorePath.isEmpty(), "keystorePath can not be empty");
-      return builder().setKeystorePath(keystorePath).build();
+      return builder().setSslConfiguration(
```

Comment:
I've modified the existing ssl config methods to delegate to the new SslConfiguration class so nothing breaks on release. Should we add a deprecation warning to these methods?